app/src/main/java/com/example/keniphhomemaintenance/dwellings/NewDwellingDialog.kt

```diff
@@ -71,7 +73,7 @@ fun NewDwellingDialog(
                     if (newAddress.value == "") {
                         newAddress.value = "Blank address. Oops!"
                     }
-                    val createdDwelling = Dwelling(
+                    val createdDwelling = Dwelling(PLACEHOLDER_ID,
```

Comment:
PLACEHOLDER_ID created to allow creation of Dwelling objects. Since Room is autoGenerating the primary key, PLACEHOLDER_ID is 0.